policyengine_us/parameters/gov/states/oh/tax/income/credits/joint_filing/other_non_refundable_credits.yaml

```diff
@@ -1,15 +1,10 @@
 description: Ohio provides a joint filing credit equal to a fraction of tax liability less these non-refundable tax credits.
 values:
   2021-01-01:
-    - oh_cdcc # Line 6
-    - oh_senior_citizen_credit # Line 3
-    - oh_retirement_credit # Line 2
-    - oh_exemption_credit # Line 9
-  2023-01-01:
-    - oh_cdcc # Line 6
-    - oh_senior_citizen_credit # Line 3
-    - oh_retirement_credit # Line 2
-    - oh_exemption_credit # Line 9
+    - oh_cdcc
```

Comment:
could you keep the Line numbers - it is easier to see which credits are meant based on the form